schematics/src/utils/registration.ts

```diff
@@ -180,6 +185,17 @@ export function addImportToFile(options: { module?: string; artifactName?: strin
   };
 }
 
+export function addLazyExportToBarrelFile(options: { path?: string; name?: string; artifactName?: string }): Rule {
```

Comment:
This rule factory does not explicitly just add "lazy" components, right? So the name should just be `addExportToBarrelFile`.